src/content/en/2020/markup.md

```diff
@@ -0,0 +1,553 @@
+---
+part_number: I
+chapter_number: 3
+title: Markup
+description: Markup chapter of the 2020 Web Almanac covering general observations, the use of elements and attributes, as well as trivia and trends.
+authors: [j9t, catalinred, iandevlin]
+reviewers: [zcorpan, matuzo, bkardell]
+analysts: [tiggerito]
+translators: []
+discuss: @@
```

Comment:
Your thread URL is https://discuss.httparchive.org/t/chapter-3-markup/2039. I'd recommend "watching" it if there's any feedback from readers.

```suggestion
discuss: 2039
```